theme/newsroom-agenda-items.tpl.php

```diff
@@ -4,28 +4,27 @@
  * @file
  * Agenda items.
  */
+  $l = variable_get('newsroom_legacy', 1);
 ?>
-<div class="date-row">
-  <?php echo $date; ?>
-  <div class="date-events">
-    <?php if (count($items) > 0): ?>
-      <?php foreach ($items as $item): ?>
-        <div class="views-row clearfix">      
-          <span class="newsroom_type"><?php echo $item->name; ?>: </span>
-          <div class="newsroom_title">
-            <?php $prefix = $item->new ? '<span class="itemFlag flagHot newItem">New</span> ' : NULL; ?>
-            <?php echo l($prefix . $item->title, $item->url, array('html' => TRUE, 'absolute' => TRUE)); ?>
+<?php if (!empty($items)): ?>
+  <div class="date-row">
```

Comment:
i see you use date-row here, but i think other contribs, even the platform itself can use it as well - highly recommend to rename it to a more specific one. also the date-events below.